modules/gis/src/lib/geo/geoarrow-metadata.ts

```diff
@@ -0,0 +1,78 @@
+// loaders.gl, MIT license
+// Copyright (c) vis.gl contributors
+
+import {Schema, Field} from '@loaders.gl/schema';
+
+export type GeoArrowEncoding =
+  | 'geoarrow.multipolygon'
+  | 'geoarrow.polygon'
+  | 'geoarrow.multilinestring'
+  | 'geoarrow.linestring'
+  | 'geoarrow.multipoint'
+  | 'geoarrow.point'
+  | 'geoarrow.wkb'
+  | 'geoarrow.wkt';
```

Comment:
I realize that `geoarrow.wkb` and `geoarrow.wkt` are inefficient, since the raw coordinates are not aligned, and we need an extra copy of the raw coordinates to compose the binary geometry format. We may need to drop the support of geoarrow.wkb/wkt.

Reply:
I already added support for these for GeoParquet so I suspect that it shouldn't be too hard to add support for them.